solr_conf/solrconfig.xml

```diff
@@ -140,7 +140,7 @@
        <str name="facet">true</str>
        <str name="facet.mincount">1</str>
        <str name="facet.limit">10</str>
-       <str name="facet.field">dataProvider_name</str>
+       <str name="facet.field">dataProvider_providedLabel</str>
```

Comment:
I think they define the default behavior for the Solr `SearchHandler`, meaning that automatically it would return `dataProvider_providedLabel` as a facet if no other facets are explicitly requested.